LibraryManagement.cpp

```diff
@@ -2,342 +2,344 @@
 #include <string>
 #include <conio.h>
 #include <cstdlib>
+#include <fstream>
 #include <ctime>
+#include <cctype>
 #include "book.h"
 #include "user.h"
 #include "bookmark.h"
 #include "Menu.h"
-
-
 using namespace std;
-
-bool isLoggedIn() {
-    static bool loggedIn = false;
-    return loggedIn;
-}
-
 Menu menu;
 Bookmark bookMark;
 
 int main() {
-	bool quit{false};
-int choose;
-	string username,password;
-    bool isPremium=false;
-    
-
-    cout<<"Enter username: ";
-    cin>>username;
-    while(password.length() < 8) {
-            cout<<"Enter password: ";
-            cin>>password;
-            if(password.length() < 8) {
-                cout<< "Password must contain 8 or more characters..\n";
-            }
-        }
-
-    char userType;
-    cout<<"Select user type(P for premium, R for regular): ";
-    cin>>userType;
-
-
-    if(userType=='P' || userType=='p') {
-    	isPremium=true;
-    	cout<<"Welcome, "<<username<<"!"<<endl;
-    	cout<<"You are a premium user."<<endl;
+	bool shouldLeaveApp = false;
+	bookMark.addDefaultBooks();
+    int choose;
+    string username,password;
+    cout << "Welcome to our program!\n";
+    cout << "[1] Login\n";
+    cout << "[2] Register\n";
+    cout << "[3] Quit\n";
+    cout << "Enter selection: ";
+    cin >> choose;
+    switch (choose) {
+        case 1:
+    	    bookMark.loginUser();
+            break;
+        case 2:
+           bookMark.registerNewUser();
+            break;
+        case 3:
+        	cout << "\nThanks for using our library management\n" ;
+            exit(0);
+            break;
+        default:
+            cout << "Wrong selection" << endl;
+            break;
+    }
+	char enteredUserType;
+    cout << "Select user type(P for premium, R for regular): ";
+    cin >> enteredUserType;
+    enteredUserType = tolower (enteredUserType);
+    if ( enteredUserType == 'p') {
```

Comment:
create a constant for `p`.  No need for spacing between ( and first character. (enteredUserType == pCharConstant)